config/management/src/lib.rs

```diff
@@ -479,40 +505,52 @@ pub mod tests {
         storage
             .create(constants::WAYPOINT, Value::String("".into()), &policy)
             .unwrap();
-
-        storage
     }
 
-    fn operator_key(namespace: &str) -> Result<Ed25519PublicKey, Error> {
```

Comment:
Can we remove the duplication between "operator_key()" and "owner_key()"? It seems the only difference is the key name, right?

Reply:
look at the top level, is it still excessively duplicated?